src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div
+        class="card__image"
+        alt="Product image"
+      ></div>
+
+      <div class="card__title">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="review-count">
+          <p>Reviews: 5</p>
+        </div>
+      </div>
+
+      <div class="card__price">
+        <div class="price-title">
+          <p>Price:</p>
+        </div>
+
+        <div class="price-value">
+          <p>$2,199</p>
+        </div>
+      </div>
+
+      <a
+        href="https://example.com"
+        class="card__button"
+        data-qa="hover"
+      >
+        Buy
+      </a>
```

Comment:
The button text 'Buy' should be more descriptive for accessibility purposes. Consider adding more descriptive text or an `aria-label` attribute to describe the action and the product it relates to.